app/src/main/java/app/revanced/integrations/settingsmenu/ReturnYouTubeDislikeSettingsFragment.java

```diff
@@ -20,6 +20,10 @@
 
 public class ReturnYouTubeDislikeSettingsFragment extends PreferenceFragment {
 
+    private static final boolean IS_SPOOFING_TO_NON_LITHO_SHORTS_PLAYER =
+            SettingsEnum.SPOOF_APP_VERSION.getBoolean()
+                    && SettingsEnum.SPOOF_APP_VERSION_TARGET.getString().compareTo("18.29.39") <= 0;
```

Comment:
If the user turns off spoofing and restarts the app, the field value will be false. Is this edge case considered?